apps/sync/js/adapters/bookmarks.js

```diff
@@ -51,12 +52,30 @@ var BookmarksHelper = (() => {
     });
   }
 
+  function removeSyncedCollectionMtime(userid) {
+    return new Promise(resolve =>
+      asyncStorage.removeItem(userid + BOOKMARKS_COLLECTION_MTIME, resolve));
+  }
+
   function getSyncedCollectionMtime(userid) {
     return new Promise(resolve => {
       asyncStorage.getItem(userid + BOOKMARKS_COLLECTION_MTIME, resolve);
     });
   }
 
+  function getLastRevisionId(userid) {
```

Comment:
You didn't address or respond to this comment https://github.com/mozilla-b2g/gaia/pull/32619#discussion_r43370675